deployment/hasura/migrations/AerieMerlin/23_activity_directive_changelog/down.sql

```diff
@@ -0,0 +1,1006 @@
+-- remove restore_activity_changelog function permissions key enum value
+drop procedure metadata.check_merge_permissions(_function metadata.function_permission_key, _permission metadata.permission, _plan_id_receiving integer, _plan_id_supplying integer, _user text);
+drop procedure metadata.check_merge_permissions(_function metadata.function_permission_key, _merge_request_id integer, hasura_session json);
+drop function metadata.raise_if_plan_merge_permission(_function metadata.function_permission_key, _permission metadata.permission);
+drop procedure metadata.check_general_permissions(_function metadata.function_permission_key, _permission metadata.permission, _plan_id integer, _user text);
+drop function metadata.get_function_permissions(_function metadata.function_permission_key, hasura_session json);
+
+drop type metadata.function_permission_key;
+create type metadata.function_permission_key
+  as enum ('apply_preset', 'branch_plan', 'create_merge_rq', 'withdraw_merge_rq', 'begin_merge', 'cancel_merge',
+    'commit_merge', 'deny_merge', 'get_conflicting_activities', 'get_non_conflicting_activities', 'set_resolution',
+    'set_resolution_bulk', 'delete_activity_subtree', 'delete_activity_subtree_bulk', 'delete_activity_reanchor_plan',
+    'delete_activity_reanchor_plan_bulk', 'delete_activity_reanchor', 'delete_activity_reanchor_bulk', 'get_plan_history');
```

Comment:
This key isn't being removed from all roles that included it. [Here](https://github.com/NASA-AMMOS/aerie/blob/6f046fb4c10b31f4210ba30a4b44eca0b61ea0ac/deployment/hasura/migrations/AerieMerlin/23_user_plan_snapshots/down.sql#L6-L8) is an example of how to do this;